File: LeetCode/1347.cpp
class Solution {
    vector<int> getCount(string &s) {
        vector<int> count(26);
        for (char c : s)
            count[c - 'a']++;
        return count;
    }
    
public:
    int minSteps(string a, string b) {
        vector<int> countA = getCount(a);
        vector<int> countB = getCount(b);
        
        int res = 0;
        for (int i = 0; i < countA.size(); i++)
            res += abs(countA[i] - countB[i]);
        
        return res / 2;
    }
};

File: EOlymp/1347.py
n = int(input())

res = 1
d = 2
while d * d <= n:
    if n % d == 0:
        count = 0
        while n % d == 0:
            count += 1
            n //= d
        res *= count + 1
    d += 1
if n > 1:
    res *= 2

print(res)
